docs/building_the_project.rst

```diff
@@ -379,6 +379,18 @@ With:
 
    -DENABLE_CURAND_BACKEND=True   \
 
+To build with the cuFFT backend instead simply replace:
```

Comment:
I agree, it could be misunderstood like that. For this PR I've tried to follow whats already here to address the linked issue, but I think this building_the_project file could do with some more love than just PRs such as this that add a few lines.

I'll update where I've contributed, since inconsistent is probably better than misleading.